src/main/java/org/apache/sysml/utils/Statistics.java

```diff
@@ -773,8 +781,9 @@ public static String display(int maxHeavyHitters)
 				sb.append("Functions recompile time:\t" + String.format("%.3f", ((double)getFunRecompileTime())/1000000000) + " sec.\n");	
 			}
 			if( ConfigurationManager.isCodegenEnabled() ) {
-				sb.append("Codegen compile (DAG,FP,CP,JC):\t" + getCodegenDAGCompile() + "/" + getCodegenFPlanCompile() 
-					+ "/" + getCodegenCPlanCompile() + "/" + getCodegenClassCompile() + ".\n");
+				sb.append("Codegen compile (DAG,FP,CP,JC):\t" + getCodegenDAGCompile() + "/" + getCodegenFPlanCompile()
+					+"[partial:"+getCodegenFPlanPartialCompile() + "]/"
+						+ getCodegenCPlanCompile() + "/" + getCodegenClassCompile() + ".\n");
```

Comment:
Maybe its a good idea to split this line into "Codegen compile (DAG, CP, JC)" and "Codegen enumerate", where the latter reports the size of the search space (number of evaluated and skipped plans), the number of evaluated plans, and the number of partially evaluated plans.